examples/hybrid/driver.jl

```diff
@@ -22,7 +22,7 @@ face_initial_condition(local_geometry) = (;)
 postprocessing(sol, output_dir) = nothing
 
 ################################################################################
-is_distributed = haskey(ENV, "CLIMACORE_DISTRIBUTED")
+is_distributed = ENV["CLIMACORE_DISTRIBUTED"] == "MPI"
```

Comment:
```suggestion
is_distributed = get(ENV, "CLIMACORE_DISTRIBUTED", "") == "MPI"
```
will work if the environment variable is not defined.